kernelci.org/content/en/blog/posts/2023/api-timeline/index.md

```diff
@@ -0,0 +1,133 @@
+Python 2.7 which stopped being maintained as a language on 1st January 2020.
+We could have just rewritten it in Python 3, which was the initial thought.
+But in the meantime, KernelCI was also growing as a project.  It wasn't just
+about building ARM kernels and doing boot testing on embedded dev boards any
+more.  It had become a Linux Foundation project aiming to test the whole
+upstream kernel.
+
+## What is this new API?
+
+Following this move, an increasing number of people became interested as it got
+under the spotlight.  That is when we started to realise that the architecture
+needed to fundamentally evolve in order to match the scale of the new mission
+it had been assigned.  The Linux kernel is a vast and complex open-source
+project with a unique ecosystem.  As such, it requires some unique tooling too.
+We all know that [Git](https://git-scm.com/) was initially created out of a
+need to manage all the kernel patches.  Now KernelCI needs an automated testing
+tool tailor-made to its unique requirements - and that's why we're finally
+launching the new API & Pipeline.
+
+It comes with lots of improvements and it's still a work-in-progress.  We'll
+keep publishing blog posts and update the documentation as things evolve over
+the next few months.  Right now we have a pipeline that can monitor Git
+repositories for new revisions, take a snapshot of the kernel source tree in a
+tarball, run KUnit with it as well as an x86 kernel build and smoke test it in
+QEMU.  It can then also send a summary email and detect regressions.  That's
+basically enough to prove we have a workable system.  Nothing too
+groundbreaking there, you might think.  So, what's all the fuss about?
+
+In a nutshell: a Pub/Sub interface to orchestrate distributed client-side
+services that can be run anywhere.  You could have your own too at home.  Also:
+user accounts so you can keep your own personal test data there, an abstraction
+for runtime environments so jobs can be run seamlessly in Docker, Kubernetes, a
+local shell, LAVA, [insert your own system here]... a new `kci` command line
+tool to rule them all and a unified Node schema to contain all the test data
+(revision, build, runtime test, regression...) in a tree or
+[DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph) to be precise.  But
+again, we'll go through all that later in more detail.  It's all based on
+requirements gathered from the community over the past few years.
+
+## Timeline
+
+The main message in this blog post is the timeline for retiring the old system
+and getting the new API in production.  Here's the proposal:
+
+Early Access
+: Monday 4th September 2023
+
+Production Deployment
+: Monday 4th December 2023
+
+Legacy System Deprecation
+: Monday 4th March 2024
+
+Legacy System Sunset
+: Monday 4th November 2024
+
+It only takes four Monday the Forth milestones to get through all this.  Here's
+what they mean:
```

Comment:
Just a minor thing.
```suggestion
It only takes four Mondays the Forth milestone to get through all this.  Here's
what they mean:
```

Reply:
I intentionally wrote it as "Monday the Forth" milestones.  There are definitely 4 milestones.  I think I'm just missing dashes in "Monday-the-Forth" to be completely correct from a grammatical point of view.

Reply:
OK I've updated this part now.